### PR TITLE
Add new major types

### PR DESCRIPTION
**Pull request by jia614**
Implemented new Major2 type :)

### CODE REVIEW

File: common/types.ts [hunk 1]
```diff
@@ -153,6 +153,109 @@ export interface INEUPrereqCourse {
...
+
+/**
+ * A Section, containing its related requirements.
+ * @param title The title of the section.
+ * @param requirements A list of the requirements within this section.
+ * @param minRequirementCount The minimum number of requirements (counts from requirements)
+ *        that are accepted for the section to be fulfilled.
+ */
+export interface Section {
+  title?: string;
```
Comment (cindy1u0) [inline common/types.ts:181]: lets make this title: string | null

**Review by cindy1u0 — APPROVED** [review]
overall looks good to meeee :))))

File: common/types.ts (hunk 1 above)
Comment (justineluo) [reply common/types.ts:181]: Let's not make this optional, since I'm pretty sure every section needs a title. I updated the docs, sorry for the confusion!

**Review by justineluo — CHANGES_REQUESTED** [review]
Just small fix, otherwise LGTM :)

**Review by justineluo — APPROVED** [review]

**Pull request merged**